index.html

```diff
@@ -20,18 +20,14 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
 
     <!-- Imports an icon to represent the document. -->
-    <link rel="icon" href="assets/icons/icon_24.png" type="image/x-icon" />
+    <link rel="icon" href="icons/icon_24.png" type="image/x-icon" />
```

Comment:
shouldn't this still be `assets\icons\icon..` since everything is in the assets folder under public?

Reply:
yet another good catch